package.json

```diff
@@ -1,12 +1,11 @@
 {
   "name": "project-require",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "description": "Allows to require project files as absolute paths, relative to the root of your node project.",
   "main": "index.js",
   "dependencies": {},
   "devDependencies": {},
   "scripts": {
-    "install": "mkdir -p node_modules/project-require && cp index.js node_modules/project-require/",
     "test": "npm install && node test.js"
```

Comment:
I have idea, I'll move that copy into test script, so it won't affect consumer installations. Will it be ok for your case? Thank you.